backend/codesirius/internal_api/views/hidden_test_bundle.py

```
@@ -0,0 +1,51 @@
+        serializer = HiddenTestBundleSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            serializer.save()
+            return CodesiriusAPIResponse(data=serializer.data)
+
+
+class HiddenTestBundleRetrieveUpdateAPIView(APIView):
+    def get_object(self, bundle_pk: int, problem_pk: int) -> HiddenTestBundle:
+        try:
+            return HiddenTestBundle.objects.get(id=bundle_pk, problem_id=problem_pk)
+        except HiddenTestBundle.DoesNotExist:
+            raise NotFound("Hidden test bundle not found")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)